schedule/yast/lvm_multipath_pvm.yaml

```diff
@@ -0,0 +1,29 @@
+---
+name:           lvm_multipath
+description:    >
+  Textmode installation test for lvm partitioning with no spearate home, on multipath with lvm validation.
+vars:
+  DESKTOP: textmode
+  MULTIPATH: 1
+  LVM: 1
+  SEPARATE_HOME: 0
+  HDDMODEL: scsi-hd
+  YUI_REST_API: 1
+schedule:
+  multipath:
+    - installation/multipath
+  guided_partitioning:
+    - installation/partitioning/select_guided_setup
+    - installation/partitioning/guided_setup/enable_lvm
+    - installation/partitioning/guided_setup/do_not_propose_separate_home
+  system_validation:
+    - console/validate_multipath
+    - console/validate_lvm
+    - console/hostname
+    - console/system_prepare
+    - console/force_scheduled_tasks
+    - shutdown/grub_set_bootargs
+    - shutdown/cleanup_before_shutdown
+    - shutdown/shutdown
+test_data:
+  <<: !include test_data/yast/multipath.yaml
```

Comment:
let's add test data via YAML_TEST_DATA instead of using include, so we set up in job group yaml configuration.

Reply:
done.